auth.c

```diff
@@ -89,6 +186,15 @@ gboolean janus_auth_check_token(const char *token) {
 	return FALSE;
 }
 
+gboolean janus_auth_check_token(const char *token) {
```

Comment:
Doesn't look like this is used anymore? I don't see anything calling this method now, `janus.c` calls `janus_auth_check_token_exists`.

Reply:
https://github.com/risetechnologies/janus-gateway/blob/janus-hmac-auth/janus.c#L696, https://github.com/risetechnologies/janus-gateway/blob/janus-hmac-auth/janus.c#L778 and https://github.com/risetechnologies/janus-gateway/blob/janus-hmac-auth/janus.c#L2615.

`janus_auth_check_token` is for checking a token against the auth API in general; `janus_auth_check_token_exists` tests only for stored-mode tokens; it is used for checks in the admin API.

now that I'm writing this though, I guess the admin API calls for tokens should be disabled completely if auth is in signed-token mode; then this distinction becomes irrelevant also.

Reply:
Yep, you're right, I was only looking at the diff... sorry for the noise!

Reply:
better thorough than buggy 😃